fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -385,14 +385,14 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
                 }
                 counterNum = ExprUtil.getIntegerConstant(counterNumExpr);
             }
-            if (k != null && k > 10000) {
-                throw new SemanticException("The maximum number of the second parameter is 10000" +
+            if (k != null && k > 100000) {
```

Comment:
Please move this hand-coding number to a place where all constant numbers are kept.

Reply:
Done